test/lit/passes/remove-unused-brs-gc.wast

```diff
@@ -11,57 +11,6 @@
   (type $struct2 (struct))
  )
 
- ;; CHECK:      (func $br_on_non_i31-1 (type $none_=>_none)
- ;; CHECK-NEXT:  (drop
- ;; CHECK-NEXT:   (block $any (result (ref null $struct))
- ;; CHECK-NEXT:    (drop
- ;; CHECK-NEXT:     (br $any
- ;; CHECK-NEXT:      (struct.new_default $struct)
- ;; CHECK-NEXT:     )
- ;; CHECK-NEXT:    )
- ;; CHECK-NEXT:    (ref.null none)
- ;; CHECK-NEXT:   )
- ;; CHECK-NEXT:  )
- ;; CHECK-NEXT: )
- (func $br_on_non_i31-1
-  (drop
-   (block $any (result anyref)
-    (drop
-     ;; An struct is not an i31, and so we should branch.
-     (br_on_non_i31 $any
-      (struct.new $struct)
-     )
-    )
-    (ref.null any)
-   )
-  )
- )
- ;; CHECK:      (func $br_on_non_i31-2 (type $none_=>_none)
- ;; CHECK-NEXT:  (drop
- ;; CHECK-NEXT:   (block $any (result nullref)
- ;; CHECK-NEXT:    (drop
- ;; CHECK-NEXT:     (i31.new
- ;; CHECK-NEXT:      (i32.const 0)
- ;; CHECK-NEXT:     )
- ;; CHECK-NEXT:    )
- ;; CHECK-NEXT:    (ref.null none)
- ;; CHECK-NEXT:   )
- ;; CHECK-NEXT:  )
- ;; CHECK-NEXT: )
- (func $br_on_non_i31-2
-  (drop
-   (block $any (result anyref)
-    (drop
-     ;; An i31 is provided here, and so we will not branch.
-     (br_on_non_i31 $any
-      (i31.new (i32.const 0))
-     )
-    )
-    (ref.null any)
-   )
-  )
- )
-
```

Comment:
Do these have replacements?

Reply:
`br_on_cast` and `br_on_cast_fail` are already well tested below, so these don't need replacements.